src/Nodes/Structures/SVGDocumentFragment.php

```diff
@@ -24,7 +25,6 @@ class SVGDocumentFragment extends SVGNodeContainer
     private $root;
     /** @var string[] $namespaces A map of custom namespaces to their URIs. */
     private $namespaces;
-
```

Comment:
Please leave empty lines as they are. If they need changing in your opinion, please submit an issue.